src/keria/app/cli/commands/start.py

```diff
@@ -108,6 +109,8 @@ def runAgent(name="ahab", base="", bran="", admin=3901, http=3902, boot=3903, co
                                 configDir=configDir,
                                 keypath=keypath,
                                 certpath=certpath,
-                                cafilepath=cafilepath))
+                                cafilepath=cafilepath,
+                                bootPassword=os.getenv("KERIA_BOOT_PASSWORD"),
```

Comment:
As we discussed, I would still suggest some indication that a more comprehensive solution should be used in production and that this is more related to test infrastructure. e.g. renaming env vars, some comments on the `authenticate` method etc

Reply:
What about `KERIA_EXPERIMENTAL_BOOT_PASSWORD` for now? That indicates it might change and/or be removed.

Reply:
Works for me!